### PR TITLE
Migrate `/textbook-beta` route and components to `/learn`

### LINKED ISSUE #2605: Redirect /learn page to new Learn page
- [x] need to redirect `/textbook-beta` to `/learn` 
- [x] need to replace `/learn` with `/textbook-beta` (reminder, clean up unused code / assets)

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes

Fixes #2605 

## Implementation details
- removed old `/learn` content, components, and assets
- renamed instances of `textbookBeta` to `learn`
- renamed instances of `TEXTBOOK_BETA` to `LEARN`
- renamed class names from `textbook-beta` to `learn`
- renamed components with `<TextbookBeta...` to `<Learn...`
- moved all `/textbook-beta` pages into `/learn`
- moved all `components/textbook-beta` to `components/learn/`
- ensured routes pointing to old `/learn` content are now pointing to updated paths

## How to read this PR

- review the UI using the branch preview
- for fuller review and to spot anything I might've missed, it's best to pull down branch and build locally

### CODE REVIEW

File: components/learn/LearnHeader.vue [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+  <header class="learn-header">
+    <div class="learn-header__container-wrapper">
+      <div class="learn-header__container">
+        <div class="learn-header__logo-container">
+          <DynamicBackgroundLogo class="learn-header__logo" />
         </div>
         <div>
-          <h1 class="textbook-beta-header__headline">
+          <h1 class="learn-header__headline">
             Qiskit Textbook (beta)
```
Comment (vabarbosa) [inline components/learn/LearnHeader.vue:10]: should we also remove the `(beta)`  text?

**Review by vabarbosa — CHANGES_REQUESTED** [review]
@techtolentino thanks again. just one minor question

File: components/learn/LearnHeader.vue (hunk 1 above)
Comment (techtolentino) [reply components/learn/LearnHeader.vue:10]: Hey @vabarbosa - not yet... if I remember, @JRussellHuffman had mentioned we'll still be in Beta for now...

Comment (vabarbosa) [reply components/learn/LearnHeader.vue:10]: good to know.

**Review by vabarbosa — APPROVED** [review]

**Comment by vabarbosa** [comment]
thanks @techtolentino

**Pull request merged**